frontend/src/styles/Team.css

```diff
@@ -0,0 +1,29 @@
+#team{
+    width: 90vw;
+    height: 80vh;
+    justify-content: center;
+    margin: 3vw 3vw 3vw;
+  }
+  
+  #team h1{
+    color: white;
+    font-weight: bold;
+    padding: 5px;
+    font-family: Arial, Helvetica, sans-serif;
+    text-align: center;
+  }
+  #team h4{
+    color: black;
+    font-weight: bold;
+    padding: 3px;
+    font-family: Arial, Helvetica, sans-serif;
+    text-align: left;
+  }
+  #team h5{
+    color: white;
+    font-weight: bold;
+    padding: 7px;
+    font-family: Arial, Helvetica, sans-serif;
+    text-align: left;
+  }
+  
```

Comment:
Maybe look into centering the bottom two cards. Other than that, I think it looks good!

Reply:
That's a good suggestion! I'll fix it :)